### DISCUSSION TITLE
repo?

### DISCUSSION THREAD

**Discussion by MAngel666** [trapexit/mergerfs#863] [Q&A]
Hi

is there a repository with el8 rpms?

TiA
Greetz

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs/releases/tag/2.32.2

Is there an issue with the packages there?

**Comment by MAngel666** [comment]
no but i would be nice to have the packages available over a repository...
For an update it would be easier to do it via package manager (yum,dnf etc)...

**Comment by trapexit** [comment]
I'm not a user of any RH based distro. I'm not familiar with the 3rd party repo situation. I could look into it for the distros that don't package recent versions of mergerfs / aren't rolling releases.

**Comment by ryecoaaron** [comment]
Bintray offers free hosting of deb and rpm repos for open source projects.

**Comment by trapexit** [comment]
I'll take a look for future releases.
